src/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration/DocumentFilters/DocumentFilterSettings.cs

```diff
@@ -11,13 +11,18 @@ namespace Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration.DocumentFilters
     public class DocumentFilterSettings
```

Comment:
yes, i will fix the nuget version.

---
In reply to: [195169752](https://github.com/Microsoft/OpenAPI.NET.CSharpAnnotations/pull/133#discussion_r195169752) [](ancestors = 195169752)

Reply:
Since its beta for 1.0.0 release, i do not want to increment major version and will keep the version as is, I think its expected for some breaking changes to be made in beta

---
In reply to: [195175227](https://github.com/Microsoft/OpenAPI.NET.CSharpAnnotations/pull/133#discussion_r195175227) [](ancestors = 195175227,195169752)